documentation/Using-the-SDK/Advanced/Creating-Your-Own-DataSource.md

```diff
@@ -0,0 +1,204 @@
+# Abstract
```

Comment:
Can we add a `Using-the-SDK/Advanced/Overview.md` that is an index of advanced topics? With a link to this doc

And can we update the "Next Steps" section of `Using-the-SDK/Creating-an-extended-table` to point to that overview file, saying view it for the more advanced features you can use? FYI there will be a merge conflict with #37, so we might want to merge that first

Reply:
I'll add one.